hooks/notification/docs/README.ArtifactHub.md

```diff
@@ -331,6 +332,82 @@ env:
+```
+Card Name: <severity>: <name>
+Card Body: <location> <category> <description> <attributes>
+```
+
+##### Requirements: 
+- Please read the [Trello API documentation](https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/) for a description on how to setup your Trello key and token.
+- Identify your target card list where the new cards will be created and get the list ID. To find the card list and label ids use the Trello JSON hack as described here https://stackoverflow.com/questions/26552278/trello-api-getting-boards-lists-cards-information
+- Read the [Trello Cards API](https://developer.atlassian.com/cloud/trello/rest/api-group-cards) for more information on the card creation options.
+
+##### Configuration
+To configure a Trello notification set the `type` to `trello` and the `endPoint` to point to your env containing the Trello cards API endpoint. You also need to define the following env vars (if an env var is not defined it will take the default value, if required and not set the notification won't be sent):
+- TRELLO_CARDS_ENDPOINT: The Trello cards API endpoint. Default: https://api.trello.com/1/cards.
+- TRELLO_KEY: Your Trello API key. Read the requirements above. Required.
+- TRELLO_TOKEN: Your Trello API token. Read the requirements above. Required.
+- TRELLO_LIST: Trello unique Id of the list where the cards will be placed. Required.
+- TRELLO_LABELS: Comma separated list of Trello label IDs to apply to the card. If empty no labels will be applied. Default: ""
+- TRELLO_POS: The position of the card in the list as defined by the Trello cards API. Options: top, bottom, or a positive float. Default: top.
+- TRELLO_TITLE_PREFIX: An optional  arbitrary text to add to the title of the card. Default "".
+
+
+##### Example Config
+
+The below example shows how to create a helm values chart and load secrets for access.
+You must have `endPoint` point to a [defined environment variable](https://github.com/secureCodeBox/secureCodeBox/blob/main/hooks/notification/hook/hook.ts#L20), not a string.
+
+```
+# cat trello_values.yaml
+
+notificationChannels:
+  - name: nmapopenports
+    type: trello
+    skipNotificationOnZeroFinding: true
+    rules:
+      - matches:
+          anyOf:
+            - category: "Open Port"
+    endPoint: TRELLO_CARDS_ENDPOINT
+env:
+    - name: TRELLO_CARDS_ENDPOINT
+      value: https://api.trello.com/1/cards
+    - name: TRELLO_KEY
+      valueFrom:
+        secretKeyRef:
+          name: trello
+          key: key
+    - name: TRELLO_TOKEN
+      valueFrom:
+        secretKeyRef:
+          name: trello
+          key: token
+    - name: TRELLO_LIST
+      value: 123a456b789c013d
+    - name: TRELLO_LABELS
+      value: "111a111b222c333f,555a666b777c888d"
+    - name: TRELLO_POS
+      value: top
+    - name: TRELLO_TITLE_PREFIX
+      value: "Alert! "
+
+# cat trello_secrets.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+    name: trello
+type: Opaque
+data:
```

Comment:
```suggestion
stringData:
```

Here's the reason why it didn't work for me. Values in "data" must be base64 encoded, while "stringData" does that conversion automatically and allows the direct insertion of the api key and token.

Alternative: Use the following command to create the secret (might be good to include in the Readme)
` kubectl create secret generic trello-secret-test --from-literal=key=YOURAPIKEY --from-literal=token=YOURTOKEN`

Reply:
Updated.